lua/spec/neotest-rspec/init_spec.lua

```diff
@@ -0,0 +1,17 @@
+local plugin = require('neotest-rspec')
```

Comment:
I'm not yet sure if this is actually loading the module from inside this project or the one installed by my plugin manager.

Reply:
I think we'll need to clone `neotest` in order to test properly. I clone plenary in my [minimal.vim](https://github.com/olimorris/persisted.nvim/blob/main/tests/minimal.vim) file for my session plugin...but something to worry about later.